static/app/components/events/contexts/utils.tsx

```diff
@@ -461,6 +461,34 @@ export function getFormattedContextData({
       }));
   }
 }
+
+function shortRuntimeVersion(version: string) {
+  // Ruby runtime version looks like:
+  // - `ruby 3.2.6 (2024-10-30 revision 63aeb018eb) [arm64-darwin23]`
+  // - `ruby 2.6.10p210 (2022-04-12 revision 67958) [universal.arm64e-darwin24]`
+  if (version.startsWith('ruby') && version.length > 25) {
+    // Extract everything from "ruby" until the first opening parenthesis
+    // This will include both the version number and any patch level
+    const match = version.match(/ruby\s+(.*?)(?:\s+\(|$)/);
```

Comment:
```suggestion
    const match = version.match(/^ruby\s+(.*?)(?:\s+\(|$)/);
```
🏌️ can restrict matches to only the start in the regexes directly but these are so small that it doesn't particularly matter